DedicatedServer/Scripts/Modes/Trackmania/FlagRush.Script.txt

```diff
@@ -501,6 +501,10 @@ Void CheckRoundEndConditions() {
 					EventFeed::SendMessage(Message, "info");
 					UIManager.UIAll.QueueMessage(C_BigMessage_Duration, 1, CUIConfig::EMessageDisplay::Big, Message, CUIConfig::EUISound::Default, 0);
 				} else {
+					// Round ends due to time out
+					declare Text Message = "$f90Time's up!";
```

Comment:
- [ ] Not a big fan of shorting like that in such messages. Could you write it out? (Time is up)
- [ ] We usually don't use exclamation marks or any punctuation in these messages (see "Overtime" above, "Round Start" and so on)
- [ ] I don't think the message should be orange. It's doesn't need special attention imo

Feel free to argue

Reply:
- that's how, in my experience, timeouts were usually written in games i played, so i just followed suit. However, it's true that it doesn't seem like how it is usually written in FlagRush. "Time is up" seems like a better option.
- some messages do have punctuation ("X got the flag!", "X scored for team Y!"). These are messages that make sense in a sentence (for example "Round Start" doesn't make sense on it's own, so it doesn't have punctuation). "Time is up" is an expression so i think it should keep punctuation.
- other time warnings are already orange, so i was feeling like it was important for consistency.

Reply:
- Your right about the punctuation part. I thought I didn't put exclamation marks for the other messages. Keep it then, but write it out.
- It's not really a warning, since the round is just over in this case. Time warning tell the player "Ok, I have to prepare for the round to end. Better hurry to turn the game in my favor". But when the time already ran out, then there's nothing the player can do about it anymore. This is just for the player to understand what happened.

Reply:
Agree with both 👍 
Seems good, I'll make these changes.